config/default/kustomization.yaml

```diff
@@ -16,6 +16,7 @@ bases:
 - ../crd
 - ../rbac
 - ../manager
+- ../prometheus
```

Comment:
- there's already a presence of `prometheus` in this manifest, please uncomment that rather than adding a duplicate

Reply:
Deleted the prometheus line from above. I will prefer all the resources to be placed together